### ISSUE TITLE
Hcaptcha

### ISSUE THREAD

**Issue by KeparYTbcc** [dessant/buster#331]
Hi i wanted to ask if it was possible to add the hcaptcha solver,
beacouse the hcaptcha accessibility thing is not accessible in private mod,
so i need an extension,

TYSM, have a great day,

```
To contact me:
Discord = Kepar#0001
Telegram =  X
Instagram = @wifi__csdl
```

- Kepar
